project_wrf.ipynb

```diff
@@ -0,0 +1,360 @@
+      "text/plain": [
+       "<Figure size 432x288 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "plt.imshow(arr[0], interpolation=\"none\")\n",
+    "plt.colorbar()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2eb4ed98-0170-4bfe-adc9-07cfb69c2580",
+   "metadata": {},
+   "source": [
+    "Error looks good - white noise! \n",
```

Comment:
nerd in me wants to see the histogram - but that's probably overkill for this application.